readme.txt

```diff
@@ -3,31 +3,38 @@ Contributors: batmoo, joen, automattic, potatomaster, albertomedina, google, xwp
 Tags: amp, mobile
 Requires at least: 4.7
 Tested up to: 5.0
-Stable tag: 0.7.2
+Stable tag: 1.0.0
 License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 Requires PHP: 5.3.6
 
-Enable Accelerated Mobile Pages (AMP) on your WordPress site.
+Enable Accelerated Mobile Pages (AMP) on your WordPress site, the WordPress way.
 
 == Description ==
 
-Bring the speed and features of the open source [AMP project](https://www.ampproject.org/) to your site, the WordPress way.
+AMP is an open-source web components library packed with highly optimized web components, a runtime system and a validation framework; together, these elements provide a well-lit path to progressive web development: the development of [beautiful and highly optimized web experiences](https://www.ampproject.org/learn/showcases/) leveraging the capabilities of the modern web. The official AMP plugin for WordPress enabling AMP content creation in WordPress seamlessly integrated with the standard content creation workflows in WordPress; and this is true both classic editing, or Gutenberg-based editing.
 
-With the plugin active, all posts on your site will have AMP-compatible versions, accessible by appending `/amp/` to the end your post URLs. For example, if your post URL is `http://example.com/2016/01/01/amp-on/`, you can access the AMP version at `http://example.com/2016/01/01/amp-on/amp/`. If you do not have [pretty permalinks](https://codex.wordpress.org/Using_Permalinks#mod_rewrite:_.22Pretty_Permalinks.22) enabled, you can do the same thing by appending `?amp=1`, i.e. `http://example.com/?p=123&amp=1`
+Some of the most salient features and capabilities provided bythe plugin are:
 
-Also, your pages and custom post types can have AMP versions. Simply check their boxes on the 'AMP Settings' page in `/wp-admin`.
+- **Compatibility tool**: to assist the development of AMP experiences by enabling AMP debugging based on exposing extensive and detailed information about validation errors that may exist, the markup/scripts causing them, and the specific components on site (e.g theme, plugin, core) bearing the responsibility of that page content.
```

Comment:
The bolded features should all have title case for consistency. Then I think the following should start with capitalized sentence. For example:

> **Compatibility Tool**: To assist the...